packages/vscode-mdx/src/extension.js

```diff
@@ -45,7 +45,7 @@ let disposable
 export async function activate(context) {
   extensions.getExtension('vscode.typescript-language-features')?.activate()
 
-  const {tsdk} = await getTsdk(context)
+  const {tsdk} = (await getTsdk(context)) ?? {tsdk: ''}
```

Comment:
Is this related to https://github.com/volarjs/volar.js/issues/122? I think it’s also related to https://github.com/mdx-js/mdx-analyzer/pull/440.

The MDX language server requires the TypeScript SDK to resolve `tsconfig.json`, wich defines things like `mdx.plugins` and `compilerOptions.jsxImportSource`.

Reply:
This is the side effect of https://github.com/volarjs/volar.js/pull/195, since a message box has already popped up by that PR when TSDK does not exist, so no action is taken here.